ci/jobs/scripts/clickhouse_proc.py

```diff
@@ -959,7 +953,8 @@ def dump_system_tables(self):
         #   [1]: https://github.com/ClickHouse/ClickHouse/issues/77320
         #
         # NOTE: we also need to override logger.level, but logger.level will not work
-        command_args_post = f"-- --zookeeper.implementation=testkeeper --logger.log=/dev/null --logger.errorlog={self.CH_LOCAL_ERR_LOG} --logger.console=1"
+        # FIXME: --logger.*s do not override config.xml settings
```

Comment:
Let's simply redirect stderr/stdout into some log then instead of using this hacks with overrides